docs/testing-code.md

```diff
@@ -55,21 +55,52 @@ For more advanced usage, you can use the npm dedicated run-scripts:
 ```shell
 # Single run in PhantomJS
 $ npm -s run test:unit
-# Watch changes and doesn't prelaunch a browser
+# Watch and run tests on change in PhantomJS
 $ npm -s run test:watch
 ```
 
 You can pass any option to karma after the `--`:
 
 ```bash
-# watch changes, with PhantomJS prelaunched
-$ npm -s run test:watch -- --browsers PhantomJS
+# Run tests a in new Chrome and Firefox instances
+$ npm -s run test:unit -- --browsers Chrome,Firefox
 # Single run with JUnit xml output
 $ npm -s run test:unit -- --reporters mocha,junit
 ```
 
+!!! note
+    If using Chrome launcher without `chrome` being on the `$PATH` (or using Chromium),
+    you need to specify the binary path by settings the environment variable
+    `CHROME_BIN`
+
 See [the official karma documentation][karma] for more details about the possible parameters.
 
+### Testing on IE
+
+You can run the test suite under Modern.ie VMs installed with either [ievms][]
+or [iectrl][] (installation is detailled on websites).
+
+```bash
+# Install IE11 under Win7 (time to have one or more coffee!)
+$ iectrl install 11
+# Run tests under IE11
+$ npm -s run test:unit -- --browsers 'IE11 - Win7'
+```
+
+!!! note
+    uData ensure compatibility for IE officialy supported by Microsoft.
+    Right now, it's IE11.
```

Comment:
Link to https://www.microsoft.com/en-us/WindowsForBusiness/End-of-IE-support?tduid=%24ad539406b325bfd230c73b988cb84242%24%24152407%24%241806294%24%24skim49X493850X5450a24d7b9886172b62caa6222a3b6b%24%24%24 ;)